### DISCUSSION TITLE
SubDyn - OpenFAST - 5MW Monopile Turbines

### DISCUSSION THREAD

**Discussion by Daeirss** [OpenFAST/openfast#2544] [Q&A]
Hey guys, I'm currently doing a project on FOWT & I'm learning to use OpenFAST, SubDyn. I've encountered a few problems such as:
- I have not yet found any datas involving SubDyn module on a 5MW Monopile Turbine OC3 in order to recalculate, compare and verify the validity of the Open Source Software (my teacher recommended me to do it).
- I don't know how to turn the results of the SubDyn's output file into a graph / diagram /.... for presentation purposes.

Thank you for reading, I hope someone can help me with these problems.

![Screenshot 2024-12-04 132942](https://github.com/user-attachments/assets/0e591504-637e-4b57-9999-63f3258057fa)

**Comment by jjonkman** [comment]
Dear @Daeirss,

The SubDyn input file that is part of the full OpenFAST model of the OC3-monopile for the NREL 5-MW baseline wind turbine is available in the OpenFAST r-test: https://github.com/OpenFAST/r-test/tree/main/glue-codes/openfast/5MW_OC3Mnpl_DLL_WTurb_WavesIrr.

You can visualize the Guyan and Craig-Bampton modes of the SubDyn model through the following website created by Emmanuel Branlard: [https://ebranlard.github.io/viz3Danim/](https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Febranlard.github.io%2Fviz3Danim%2F&data=05%7C01%7Cjason.jonkman%40nrel.gov%7C428af22ec2fb41a957d408db2c6969e2%7Ca0f29d7e28cd4f5484427885aee7c080%7C0%7C0%7C638152603494424515%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=2XFR1A6O3aoXOkbkONS2ZdYLes00p5Osa1QerTp8fWo%3D&reserved=0); to use this feature, simply load the _.json_ summary file generated by SubDyn.

Best regards,
